catalogue/webapp/pages/search/index.tsx

```diff
@@ -56,44 +89,99 @@ export const SearchPage: NextPageWithLayout<Props> = ({
+        },
+      }}
+      icon={arrow}
+      isIconAfter={true}
+      colors={{
+        border: 'yellow',
+        background: 'yellow',
+        text: 'black',
+      }}
+      hoverUnderline={true}
+    />
+  );
+
   return (
-    <div className="container">
-      <Space v={{ size: 'l', properties: ['margin-top', 'margin-bottom'] }}>
+    <main>
+      <Space v={{ size: 'l', properties: ['margin-bottom'] }}>
         {!stories && !images && !works ? (
-          <SearchNoResults query={queryString} hasFilters={false} />
+          <div className="container">
+            <SearchNoResults query={queryString} hasFilters={false} />
+          </div>
         ) : (
-          <main>
-            <pre
-              style={{
-                fontSize: '14px',
-                overflow: 'hidden',
-              }}
-            >
-              {stories && (
-                <details>
-                  <summary>STORIES</summary>
-                  {JSON.stringify(stories, null, 1)}
-                </details>
-              )}
-
-              {images && (
-                <details>
-                  <summary>IMAGES</summary>
-                  {JSON.stringify(images, null, 1)}
-                </details>
-              )}
-
-              {works && (
-                <details>
-                  <summary>WORKS</summary>
-                  {JSON.stringify(works, null, 1)}
-                </details>
-              )}
-            </pre>
-          </main>
+          <>
+            {stories && (
+              <StoriesSection as="section">
+                <div className="container">
+                  <SectionTitle sectionName="Stories" />
+
+                  <StoriesGrid stories={stories} />
+
+                  <Space v={{ size: 'l', properties: ['padding-top'] }}>
+                    <SeeMoreButton
+                      text={`All stories`}
+                      pathname="/search/stories"
+                    />
+                  </Space>
+                </div>
+              </StoriesSection>
+            )}
+
+            {images && (
+              <ImagesSection>
+                <div className="container">
+                  <SectionTitle sectionName="Images" />
+
+                  <ImageEndpointSearchResults images={images} />
+
+                  <Space v={{ size: 'l', properties: ['padding-top'] }}>
+                    <SeeMoreButton
+                      text={`All images`}
+                      pathname="/search/images"
+                    />
+                  </Space>
+                </div>
+              </ImagesSection>
+            )}
+
+            {works && (
+              <WorksSection>
+                <div className="container">
+                  <SectionTitle sectionName="Catalogue" />
+
+                  <WorksSearchResults works={works} />
+
+                  <Space v={{ size: 'l', properties: ['padding-top'] }}>
+                    <SeeMoreButton
+                      text={`All catalogue`}
```

Comment:
Asked for proper copy here https://github.com/wellcomecollection/wellcomecollection.org/issues/8741#issuecomment-1375937663